eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/mob/ButcherCommand.java

```diff
@@ -41,36 +41,17 @@ void execute(@Context Player player) {
 
     @Execute
     @DescriptionDocs(description = "Kills all mobs in specified chunks around you", arguments = "<chunks>")
-    void execute(@Context Player player, @Arg(ButcherArgument.KEY) int chunks) {
+    void execute(@Context Player player, @Arg(ButcherArgument.KEY) Integer chunks) {
```

Comment:
why not int?

Reply:
Because ButcherArgument is for Integer type.

Reply:
Set int.class type in annotation